### PR TITLE
FIREFOX and CHrome ONLY- Thin black box displays around screenshots #572

### PR DESCRIPTION
**Pull request by rkenyon1969**
@clarkjefcoat 
@wski 
@SOAP1914

### CODE REVIEW

**Comment by rkenyon1969** [comment]
thumbs anyone?

@clarkjefcoat @wski @SOAP1914

**Comment by clarkjefcoat** [comment]
:+1:

**Comment by SOAP1914** [comment]
👍 Mac's VoiceOver still works with this change

**Comment by SOAP1914** [comment]
@rkenyon1969

File: app/styles/main.scss [hunk 1]
```diff
@@ -53,6 +53,14 @@ strong {
   clear: both;
 }
 
+[tabindex] {
+   outline: none !important;
+}
```
Comment (wski) [inline app/styles/main.scss]: Not sure we want this one, there are things with a tabindex that we wan't highlighted. For instance the button in the notifications modal

**Comment by wski** [comment]
👎 small change needed.

**Comment by SOAP1914** [comment]
@rkenyon1969 @wski @clarkjefcoat  So I can still see the highlight in the top nav, which is what we want. But @rkenyon1969 was only touching the ones in popup modal Review tab (right?)

**Comment by SOAP1914** [comment]
screen shot 2016-04-25 at 4 44 03 pm

**Comment by wski** [comment]
This solution will work fine as long as we never add a tabindex to a form input

**Comment by SOAP1914** [comment]
in the future we will need it for the form inputs @wski @rkenyon1969 @clarkjefcoat

**Comment by wski** [comment]
nvm, i read it wrong 👍

**Comment by wski** [comment]
nevermind x2, you will need to add an exception for `select` and `textarea` 👎

**Comment by rkenyon1969** [comment]
Thank you all for your help. Let's try attempt part 2:

/*[tabindex] {
   outline: none !important;
   outline: 'default';
}

[tabindex][input] {
   outline: none !important;
}*/

div:focus{
 outline: none !important;
}

section:focus{
 outline: none !important;
}

ul:focus{
 outline: none !important;
}

**Comment by bhalt** [comment]
I would stay away from using `!important` in our css as that quickly gets dangerous. I would also hesitate to override any form element behavior as it is currently set up in bootstrap/browser defaults -- since that adds both another layer of complexity to future development/compliance as well as more CSS to manage.

To solve this display issue, I think it would be as simple as adding 

[tabindex] { outline: 0; }

to the global CSS. This would remove the outline from anything that normally wouldn't have a focus outline but is inheriting one from the tabindex attribute (divs, etc) -- but leave the normal required behavior for form elements.

I haven't extensively tested this (just in Chrome) so it could use more testing to make sure it solves the issue. @clarkjefcoat

**Comment by rkenyon1969** [comment]
@clarkjefcoat @SOAP1914 @wski @bhalt 

Question: Should we remove the outline? Is there a user/508 benefit leaving it alone. Browsers implement outline to allow users to visually determine where they are on a page.

Question: What if we standardized so that all outlines are the same; maybe with a unique(cool) design:

:focus {
 outline-style: dotted;
outline-color: #cc0000;
}

**Comment by SOAP1914** [comment]
I'll have to see it first @rkenyon1969 (the unique design)it more of a nice to have. 
@clarkjefcoat @bhalt @wski what are your thoughts on this?

**Comment by wski** [comment]
@bethany the reason he can't do just `[tabindex] { outline: 0; }` is be cause form elements need to have that outline, and some form inputs have `tabindex` thus there needs to be exceptions so when you're tabbing through a page, you know where the cursor is at.

**Comment by wski** [comment]
halt

**Comment by wski** [comment]
@rkenyon1969 making a standard outline might be more effort than it's worth because some browsers don't even allow you to remove it from things that have outline by default. (many mobile browsers and IE)

**Comment by bhalt** [comment]
@wski When I added `[tabindex] { outline: 0; }` it didn't overwrite any form element focus displays. None of the current form elements in bootstrap get their outline from the tabindex attribute property, so that shouldn't change any form behavior, as long as the `!important` isn't added. Can you give more information if you tested it and found otherwise?

@rkenyon1969 we actually do customize the outline color in the OZP bootstrap but like Matt said, depending on the browser (and element that the element is on), the focus property will be interpreted different ways. If we make style changes, that should probably be a different PR than this bug fix.

**Comment by rkenyon1969** [comment]
@clarkjefcoat @wski @bhalt @SOAP1914 

Ok. Final solution (please thumbs up):

[tabindex] {
   outline: none;
}

**Comment by akonsowski** [comment]
:+1:

**Comment by rkenyon1969** [comment]
@clarkjefcoat @wski @bhalt - I'd like to close this today if possible. Please thumbs up if you think it is ready. - thank you!

**Comment by clarkjefcoat** [comment]
:+1:

**Pull request merged**